### DISCUSSION TITLE
Open default help when clicking questionmark

### DISCUSSION THREAD

**Discussion by Caeylinity** [NilsSve/DfRefactor#25] [Ideas]
The 'Help' icon lights up when you hover it and shows a tooltip, but clicking it does nothing. I'm not sure if this is intended in the first place, but assuming it is, maybe opening the local help by default would be nice.

**Comment by NilsSve** [comment]
The behavior has been changed. Clicking the drop-down button now activates the local help.

Originally it had been designed to do nothing when the drop-down was clicked to "encourage" the user to see the drop-down menu options. But I can see how that can be confusing and thus it was changed.
